Gemfile

```diff
@@ -27,13 +27,13 @@ gem 'local_time', '>= 2.0.0'
 gem 'turbolinks', '~> 5'
 
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.5'
+gem 'jbuilder', '~> 2.7'
 
-# Use ActiveStorage variant
-# gem 'mini_magick', '~> 4.8'
+# Use Active Storage variant
+# gem 'image_processing', '~> 1.2'
```

Comment:
Can these be removed?

Reply:
They can but we left them there as a nice reminder that someday we should probably move out propitiatory attachment management over to Active Storage 😄